.gitignore

```diff
@@ -4,3 +4,17 @@
 ./testing/alg-theme-cala-config/alg-*
 
 ./testing/calamares-3.3.5/calamares*
+
+# Ignore from cala-configs
+./cala-configs/alg-wm-cala-config/objects/pack/*
+
+# Ignore from calamares
+./calamares/calamares-3.2.26
+./calamares/calamares-3.2.39
+./calamares/calamares-3.2.47
+./calamares/calamares-3.2.62
+./calamares/calamares-3.3.5
+
+*.tar
```

Comment:
i think since the .gitignore is in root, it might not ignore the tarballs in the subfolders (requires testing)